pkg/virt-handler/heartbeat/ksm_test.go

```diff
@@ -184,7 +187,7 @@ var _ = Describe("KSM", func() {
 			fakeClient := fake.NewSimpleClientset(node)
 			heartbeat := NewHeartBeat(fakeClient.CoreV1(), deviceController(true), clusterConfig, "mynode")
 
-			heartbeat.do()
+			heartbeat.handleKSMForceUpdate()
```

Comment:
I think that here and in most cases we should test with `handleKSM()`, leaving the usage of `handleKSMForceUpdate()` to edge cases (actually it is only used once at the beginning).
Would you mind adjusting this and, if required, the node labels?

Reply:
Good point!

What I did is wrapped `newHeartbeat` in a way that it's both easier to read and write (since most of the configs are the same all over), and that executes `handleKSMForceUpdate()` once at the time of the heartbeat's creation, to simulate a first forced run.

Let me know what you think :)

Reply:
Makes sense, thank you! But some failures occur :)

Reply:
Fixed :)
I've tried to use `NewClientset` since `NewSimpleClientset` is deprecated, but it causes failures for some reason. Let's defer this to a follow-up :)